### DISCUSSION TITLE
Upgrading to LLVM 18?

### DISCUSSION THREAD

**Discussion by cement-drinker** [dslm4515/CMLFS#103] [Ideas]
With chimera Linux's patches, it's possible to build LLVM 18.1.8 in stage1.

**Comment by cement-drinker** [comment]
> With chimera Linux's patches, it's possible to build LLVM 18.1.8 in stage1. 

Nevermind i have been Ripping my hair out for a day tryna do that. Stage 3 is my next bet

**Comment by dslm4515** [comment]
My MLFS repo is currently out of date, so I am updating it at the moment.
My goal is to have MLFS & CMLFS updated to the same package versions to make it easier to update each repo.
